infrastructure/static/js/mm-webflow.js

```diff
@@ -847,6 +847,11 @@ function mmWebflow(js) {
 
         $(".detail-button_wrapper").hide();
         $(".subsection-chart__detail").hide();
+
+        var summaryYear = js.summary_year
+        if (summaryYear.split('/')[0] > implementYear.split('/')[0]) {
```

Comment:
you don't need the split to do a valid comparison, do you?

Reply:
Interesting, so 2019/2020 is run as a division operation. But that should be okay because successive periods will always be slightly greater than previous ones. Removed the splits